controllers/csiaddonsnode_controller.go

```diff
@@ -18,19 +18,31 @@ package controllers
 
 import (
 	"context"
+	"errors"
+	"fmt"
 
+	csiaddonsv1alpha1 "github.com/csi-addons/kubernetes-csi-addons/api/v1alpha1"
+	"github.com/csi-addons/kubernetes-csi-addons/internal/connection"
+	"github.com/csi-addons/kubernetes-csi-addons/internal/util"
+
+	"github.com/go-logr/logr"
+	apiErrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
nit: convention is to use lowercase for imports

Reply:
fixed this too, PTAL